### PR TITLE
Fixing sample usage tests

### PR DESCRIPTION
**Pull request by pdolega**
Fix for the issue described here: https://github.com/joscha/play-authenticate/pull/298#issuecomment-205277565

### CODE REVIEW

File: samples/java/play-authenticate-usage/test/test/OAuth2Test.java [hunk 1]
```diff
@@ -44,16 +42,11 @@ protected Application provideApplication() {
 
         amendConfiguration(additionalConfiguration);
```
Comment (pdolega) [inline samples/java/play-authenticate-usage/test/test/OAuth2Test.java]: Removing this section is not related to the issue (it was not used obviously) - just found it while fixing `additionalConfiguration`.

File: samples/java/play-authenticate-usage/test/test/OAuth2Test.java [hunk 2]
```diff
@@ -44,16 +42,11 @@ protected Application provideApplication() {
...
-        additionalPlugins.add(MyTestUserServiceService.class.getName());
-        additionalPlugins.add(getProviderClass().getName());
-        amendPlugins(additionalPlugins);
-
-
         return new GuiceApplicationBuilder()
                 .bindings(
                         bind(getProviderClass()).toSelf().eagerly()
                 )
+                .configure(additionalConfiguration)
```
Comment (pdolega) [inline samples/java/play-authenticate-usage/test/test/OAuth2Test.java]: this was lacking line

**Comment by pdolega** [comment]
@joscha ^^ I think this might just fix test failures that prevent 0.8 from being published.

**Comment by joscha** [comment]
Thanks @pdolega - there are still some selenium problems, however probably not related to the upgrade - the weird thing is, however that I am not able to run the tests locally any more - have you tried running them with the command provided in #298?

**Comment by joscha** [comment]
got the tests working. fixed one of them, one is still broken and needs a bit of selenium-massaging.

**Comment by joscha** [comment]
all green - `0.8.0-SNAPSHOT` has been released.

**Comment by pdolega** [comment]
Sorry for late answer @joscha 

> Thanks @pdolega - there are still some selenium problems, however probably not related to the upgrade - the weird thing is, however that I am not able to run the tests locally any more - have you tried running them with the command provided in #298?

I did manage to run them (saw browser being fired), saw Google authentication being performed successfully (only one I tried end 2 end - with my actual credentials). As for the rest I only verified that browser appeared, incorrect credentials have been specified (basically XXX everywhere) and at that point it obviously failed.

My mistake - thought it would be sufficient.

**Comment by joscha** [comment]
All good no worries, thank you very much for your contrib. Let me know if `0.8.1-SNAPSHOT` works for you!
